docs/installation.rst

```diff
@@ -269,6 +269,25 @@ Also make sure you have musl-gcc in your path::
 Then just build using cargo and the appropriate target::
 
     $ cargo build --target x86_64-unknown-linux-musl
+    
+Build from source (Gentoo, docker)
+==========================
+
+Clone vagga repository
```

Comment:
I believe all these needs double-colon for proper formatting

Reply:
fixed